driver-mapping/src/main/java/com/datastax/driver/mapping/AnnotationParser.java

```diff
@@ -194,6 +194,9 @@ else if (propertyMapper.isClusteringColumn())
                 throw new IllegalArgumentException(String.format("Field %s does not exist in type %s.%s",
                         propertyMapper.columnName, ksName, userType.getTypeName()));
 
+            for (Class<?> fieldUdt : TypeMappings.findUDTs(field.getGenericType()))
```

Comment:
Not sure if this is the right fix, but it does work.  Alternatively, `TypeMappings.findUDTs` could recurse nested UDTs like it does for Collections, wdyt?